### ISSUE TITLE
deadline for final document?

### ISSUE THREAD

**Issue by taylori** [cgrandin/hake-assessment#417]
I don't remember a hard deadline for any post-SRG edits to the assessment document, but I think it's typically been completed sometime during the week following the SRG. However, this year for the first time the JMC meeting immediately follows the SRG. Do we know whether the final document is needed prior to the JMC? This might be a question for the JMC co-chairs.

**Comment by andrew-edwards** [comment]
The schedule has the SRG report being available on 2nd March (next Friday), with no mention of our assessment. Presumably we can't be expected to do any (substantial) revisions by then also - I'm fairly sure we haven't in recent years. Sounds like the SRG report contains enough info (plus the submitted assessment) for the JMC. Obviously we can do any edits during the SRG meeting, but it's still beneficial to then leave it and have a final read over the following week (I can do that). 

I just downloaded the NOAA-posted version of what we submitted - it is 11.583 Mb, and the one we submitted (and on Google Drive) is 11.573 Mb. The slight increase is likely due to some web accessibility things. A quick scroll through and all looks fine so I expect it's only something minor that's changed. I didn't keep track of how long it took to get posted on the website after we submitted, but I think there was at least a day delay due to the web accessibility. So even if we get it finished quickly it might not be posted straight away (but could get distributed).
